web/src/engine/websites/MangaBat.ts

```diff
@@ -0,0 +1,22 @@
+import { Tags } from '../Tags';
+import icon from './MangaBat.webp';
+import { DecoratableMangaScraper } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import * as MangaNel from './decorators/MangaNel';
+
+@MangaNel.MangaCSS(/mangabat\.com/)

```

Comment:
Should the regex be more strict e.g., `/https?:\/\/\w*\.?mangabat\.com\/read-/`?